src/cond.d

```diff
@@ -410,6 +410,9 @@ extern (C++) final class VersionCondition : DVCondition
         if (!global.params.versionids)
             global.params.versionids = new Strings();
         global.params.versionids.push(cast(char*)ident);
+        import core.stdc.stdio;
+        if (global.params.verbose)
+            printf("version=\t%s\n", cast(char*)ident);
```

Comment:
The cast assumes ident is a zero-terminated string. Since the code works, it seems things just happen to be so. But just to make sure we don't depend on a non-local happenstance, let's do this:

`printf("version=\t%-*s\n", cast(int) ident.length, ident.ptr);`